website-custom-views/src/content/development/permissions.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Permissions
+---
+
+As we learned in [OAuth Scopes and user permissions](/concepts/oauth-scopes-and-user-permissions), each Custom View has a unique pair of default user permissions at its disposal: "view" and "manage." Additionally, you can use more granular permission groups to cover strict business requirements.
+
+When developing a Custom View you might want to enforce these user permissions in some parts of your application. For example, performing certain actions like creating, updating, or deleting a resource should only be possible if the user has "manage" permission.
+
+# Defining constants
+
+The user permissions can be computed using the `resolveCustomViewResourceAccesses` function, to avoid defining them manually. We recommend to define them in a `constants.js` file.
+
+```js title="constants.js"
+import { resolveCustomViewResourceAccesses } from '@commercetools-frontend/application-config';
+
+export const entryPointUriPath = 'channels';
```

Comment:
For Custom Views this is not needed.

```suggestion
```

Reply:
Update here: 558993f8b1746e3bd9f8435c3f961a39c118c7df